Sources/NibLoader.swift

```diff
@@ -71,6 +73,28 @@ public struct IBNibLoader<NibLoadableView: NibLoadable> where NibLoadableView: U
         nibView.isHidden = view.isHidden
     }
     
+    private func copyConstraints(to nibView: UIView) {
+        let attributesToCopy: [NSLayoutConstraint.Attribute] = [.height, .width]
+        let constraintsToCopy = view.constraints.filter {
+            return attributesToCopy.contains($0.firstAttribute)
+        }
+        for constraint in constraintsToCopy {
+            var secondItem: Any? = nil
+            if attributesToCopy.contains(constraint.secondAttribute) {
+                secondItem = nibView
```

Comment:
It is not obvious wht this isn't the `contstraint.toItem` 🤔

Reply:
For .height and .width constraint, there is only ```item``` (aka firstItem), but if you add aspect ratio from IB then the toItem (aka secondItem) should be set to the same view as firstItem - the secondAttribute is then .height or .width too